bin/main.rb

```diff
@@ -1,109 +1,34 @@
 #!/usr/bin/env ruby
-class Game
-  attr_accessor :tic_tac_toe, :player1, :player2
-
-  def initialize
-    @tic_tac_toe = Board.new
-    puts %(First player; please type your name!)
-    @player1 = Player.new(gets.chomp, 'X')
-    puts %(Second player; please type your name!)
-    @player2 = Player.new(gets.chomp, 'O')
-  end
-
-  def welcome
-    puts %(Welcome on board #{@player1.name} and #{@player2.name}, let the Tic-Tac-Toe game begins!)
-    puts %(Player 1: #{@player1.name} has token: #{@player1.token})
-    puts %(Player 2: #{@player2.name} has token: #{@player2.token})
-  end
-
-  def play
-    welcome
-    loop do
-      @tic_tac_toe.ask_move(@player1)
-      @tic_tac_toe.mark_move(@player1)
-      @tic_tac_toe.show_board
-      @tic_tac_toe.draw?(player1)
-      @tic_tac_toe.winner?(@player2)
-      @tic_tac_toe.ask_move(@player2)
-      @tic_tac_toe.mark_move(@player2)
-      @tic_tac_toe.show_board
-      @tic_tac_toe.draw?(player2)
-      @tic_tac_toe.winner?(@player2)
-    end
+# rubocop: disable Style/Next
+require_relative '../lib/game.rb'
+
+puts %(First player; please type your name!)
+player1 = Player.new(gets.chomp, 'X')
+puts %(Second player; please type your name!)
+player2 = Player.new(gets.chomp, 'O')
+TicTacToe = Game.new(player1, player2)
+puts %(Welcome #{TicTacToe.player1.name}! Your token is: #{TicTacToe.player1.token})
+puts %(Welcome #{TicTacToe.player2.name}! Your token is: #{TicTacToe.player2.token})
+
+loop do
+  puts %(#{TicTacToe.player1.name}'s turn. Type a number between 1 and 9: )
+  TicTacToe.player1.next_move = gets.chomp.to_i
+  TicTacToe.mark_board(player1)
+  TicTacToe.board_tic.show_board
+  if TicTacToe.winner?(player1) || TicTacToe.draw?(player1)
+    puts %(Congratulations #{player1.name}, you won!) if TicTacToe.winner?(player1)
+    puts %(It's a draw!) if TicTacToe.draw?(player1)
+    break
```

Comment:
You should create a method to store this code. 
I suggest having one for the winner and one for the draw,

TIP: you don't need to check for a winner until the **fifth** move. You only need to check for a draw on the **ninth** move if there's no winner.